### PR TITLE
QoH Arcana Beats + Death Counters

### PR DESCRIPTION
**Pull request by PotatoTomahto**
## About The Pull Request

QoH has Arcana Beats in passive breach
Big Bird needs 2 deaths to drop a qliphoth, MOSB needs 3 deaths
Funeral and QoH should be able to target people in melee range now
Some more apoca bird tweaks

## Why It's Good For The Game
Finishes QoH's attacks and tweaks some death counters so they actually have an effect.

## Changelog
:cl:
add: arcana beats
tweak: death counters
tweak: apoca bird
:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/waw/hatred_queen.dm [hunk 1]
```diff
@@ -198,6 +217,70 @@
...
+	if(beats_cooldown > world.time)
+		return FALSE
+	if(!can_act)
+		return FALSE
+	can_act = FALSE
+	if(target)
+		face_atom(target)
+	icon_state = "hatredbeats"
+	visible_message("<span class='danger'>[src] prepares to mark the enemies of justice!</span>")
+	var/turf/target_turf = get_ranged_target_turf_direct(src, target, 14)
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/waw/hatred_queen.dm:230]: That's too far. Arcana beats are meant to be a melee attack.

File: code/modules/mob/living/simple_animal/abnormality/waw/hatred_queen.dm [hunk 2]
```diff
@@ -198,6 +217,70 @@
...
+	var/list/turfs_to_hit = getline(src, target_turf)
+	playsound(src, 'sound/abnormalities/hatredqueen/gun.ogg', 65, FALSE, 10)
+	var/obj/effect/qoh_sygil/S = new(get_turf(src))
+	S.icon_state = "qoh1"
+	addtimer(CALLBACK(src, .atom/movable/proc/say, "Go! Arcana Beats~!"))
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/waw/hatred_queen.dm:235]: > // FUCK YOUR TILDES

File: code/modules/mob/living/simple_animal/abnormality/waw/hatred_queen.dm [hunk 3]
```diff
@@ -198,6 +217,70 @@
...
+	if(!can_act)
+		return FALSE
+	can_act = FALSE
+	if(target)
+		face_atom(target)
+	icon_state = "hatredbeats"
+	visible_message("<span class='danger'>[src] prepares to mark the enemies of justice!</span>")
+	var/turf/target_turf = get_ranged_target_turf_direct(src, target, 14)
+	var/list/turfs_to_hit = getline(src, target_turf)
+	playsound(src, 'sound/abnormalities/hatredqueen/gun.ogg', 65, FALSE, 10)
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/waw/hatred_queen.dm:232]: You might want to move this to the actual damage part.

File: code/modules/paperwork/records/info/waw.dm [hunk 4]
```diff
@@ -83,7 +83,7 @@
 	Work Damage : High	<br>
 	- When the work result was Good, the Qliphoth Counter increased.	<br>
 	- When the work result was Bad, the Qliphoth Counter lowered.	<br>
-	- When 3 employees died, the Qliphoth Counter lowered.	<br>
+	- When 2 employees died, the Qliphoth Counter lowered.	<br>
```
Comment (EgorDinamit) [inline code/modules/paperwork/records/info/waw.dm:86]: Do the same change for MoSB paper.

**Comment by dsmith328** [comment]
For the love of Lobotomy squash commits.

**Pull request merged**